Src/Screens/OfferScreen/Screens/PresentStartDate.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  PresentStartDate.swift
+//  test
+//
+//  Created by Pavel Barros Quintanilla on 2020-01-14.
+//
+
+import Foundation
+import Flow
+import Presentation
+import UIKit
+import SnapKit
+import Apollo
+
+struct PresentStartDate {
```

Comment:
Call this `ChooseStartDate` or something similar instead of `Present*`. We understand that it is a Presentable from it implementing the protocol already.